docker-compose.yml

```diff
@@ -16,6 +16,15 @@ services:
     networks:
       - frontend
       - backend
+  mailhog:
+    image: mailhog/mailhog:latest
+    ports: 
+      - 1025:1025
```

Comment:
I don't think we should be exposing this on the host, as I don't think we need this from the host's perspective.

Reply:
You're right, I'll remove it